src/popup/components/SeedPhraseNotification.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div :class="['seed-phrase-notification', { error: hasError }]">
+    <div class="icon-wrapper">
+      <Alert v-if="hasError" />
+      <CheckCircle v-else />
+    </div>
+    <div class="text">
+      <div>
+        {{
+          hasError
+            ? $t('pages.seed-phrase-settings.seed-phrase-incorrect')
+            : $t('pages.seed-phrase-settings.seed-phrase-correct')
+        }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api/dist/vue-composition-api';
+import CheckCircle from '../../icons/check-circle.svg?vue-component';
+import Alert from '../../icons/alert.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    CheckCircle,
+    Alert,
+  },
+  props: {
+    hasError: Boolean,
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.seed-phrase-notification {
+  min-height: 176px;
+  border: 2px solid variables.$color-success-dark;
+  border-radius: 16px;
```

Comment:
I suggest to create variable for this as this is the same value as the AccountCard has. I talked with Tsvetan and he told me that this radius is a common thing for large cards.